src/store/conversations/conversations.epics.ts

```diff
@@ -1542,6 +1553,11 @@ const publishFolderEpic: AppEpic = (action$, state$) =>
                 : folder.publishVersion,
           }));
 
+        const temporaryFolders = getTemporaryFoldersToPublish(
+          publishedAndTemporaryFolders,
+          newFolders[newFolders.length - 1].folderId,
```

Comment:
Are you sure that correct folder will always be on last place i.e. the order is correct?